components/server/profile.tsx

```diff
@@ -0,0 +1,192 @@
+import React from "react";
+import { Layout, LayoutContent } from "@uoguelph/react-components/layout";
+import { Footer } from "@uoguelph/react-components/footer";
+import { Header } from "@/components/server/header";
+import { getProfileContent } from "@/data/drupal/profile";
+import { HtmlParser } from "@/components/client/html-parser";
+import { Link } from "@uoguelph/react-components/link";
+import { notFound } from "next/navigation";
+import { Breadcrumbs } from "@/components/server/breadcrumbs";
+import Image from "next/image";
+import { Container } from "@uoguelph/react-components/container";
+import { Typography } from "@uoguelph/react-components/typography";
+import { LdapContactInfo } from "@/components/server/ldap-contact-info";
+import { getIconForUrl } from "@/lib/ug-utils";
+import { 
+  UniwebAffiliations,
+  UniwebCurrentTeaching,
+  UniwebDegrees,
+  UniwebPublications,
+  UniwebResearchDesc,
+  UniwebResearchInterests 
+} from "@/components/server/uniweb-components";
+import { FullProfile } from "@/lib/types/profile";
+
+// Component-specific interface that extends the shared FullProfile type
+interface ProfileContent extends FullProfile {
+  // Additional fields specific to the server component can be added here
+}
+
+export type ProfileProps = {
+  id: string;
+  pre?: React.ReactNode;
+  post?: React.ReactNode;
+};
+
+export async function Profile({ id, pre, post }: ProfileProps) {
+  const content = await getProfileContent(id) as ProfileContent | null;
+
+  // Couldn't fetch content for this id.
+  if (!content) {
+    if (process.env.NODE_ENV === "development") {
+      console.warn(`Couldn't find content for profile with id ${id}`);
+
+      if (process.env.ALWAYS_SHOW_PUBLISHED_CONTENT) {
+        console.warn("It's possible this profile is not published. ALWAYS_SHOW_PUBLISHED_CONTENT is set to true.");
+      }
+    }
+
+    notFound();
+  }
+
+  return (
+    <Layout>
+      <Header name={content.primaryNavigation?.menuName?.toUpperCase().replaceAll("-", "_")}></Header>
+
+      <LayoutContent container={false}>
+        <Breadcrumbs url={content.path ?? undefined} />
+        <Container>
+          <Typography type="h1" as="h1" className="mb-4">
+            {content?.title}       
+          </Typography>
+          <div className="md:flex md:flex-row-reverse md:gap-6">
+            <div className="md:flex-1">              
+              {pre && pre}
+              {content.profileJobTitle && <Typography type="h3" as="p">{content.profileJobTitle}</Typography>}

```

Comment:
Could we remove the top margin above job title?
`{content.profileJobTitle && <Typography type="h3" as="p" className="mt-0">{content.profileJobTitle}</Typography>}`

Makes it:
<img width="1410" height="200" alt="image" src="https://github.com/user-attachments/assets/5b808984-86f3-4707-9e20-c112a477e4b0" />

Instead of: 
<img width="1477" height="245" alt="image" src="https://github.com/user-attachments/assets/b6c6947e-45e6-4307-9a48-0d490f9308b1" />